modules/map/hooks/useNormalizedPoint.ts

```diff
@@ -0,0 +1,19 @@
+import { useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { NormalizedPoint, SelectedPoint } from '@/modules/map/types'
+import { normalizePoint } from '@/modules/map/utils/normalizePoint'
+
+export function useNormalizedPoint<Point extends SelectedPoint | undefined | null>(
+  point: Point,
+): Point extends undefined | null ? NormalizedPoint | null : NormalizedPoint
+export function useNormalizedPoint(
+  point: SelectedPoint | undefined | null,
+): NormalizedPoint | null {
+  const [, i18n] = useTranslation()
```

Comment:
You can also use `useLocale()` custom hook we have: `const locale = useLocale()`. This is also how other intl libraries work. 
Or if you need i18n instance, we usually do `const { i18n } = useTranslation()`